tests/dummy/app/components/clazz-basic/component.js

```diff
@@ -0,0 +1,19 @@
+import Component from '@ember/component';
+import { connect } from 'ember-redux';
+
+const stateToComputed = state => ({
+  low: state.low
+});
+
+const dispatchToActions = dispatch => ({
+  up: () => dispatch({type: 'UP'})
+});
+
+class MyClazz extends Component {
+  constructor() {
+    super(...arguments);
+    this.color = 'orange';
+  }
+}
+
+export default connect(stateToComputed, dispatchToActions)(MyClazz);
```

Comment:
Long term this would probably be nice as a class decorator:

```js
@connect(stateToComputed, dispatchToActions)
export default class MyClazz extends Component {

}
```

But I think redux has a lot of experimenting it can do with decorators in general, and they still aren't 100% complete